docs/advanced-topics/verification.rst

```diff
@@ -0,0 +1,190 @@
+Verification
+============
+
+What is a verified tool/workflow?
+---------------------------------
+A verified tool/workflow indicates that it was successfully run and verified by either:
+
+-  the :ref:`Dockstore ToolTester`
+-  our team
+-  an outside party
+
+Historically, the majority of tool validation has been done by the docktesters
+team currently headed by Miguel Vazquez and formerly headed by Francis
+Ouellette.
+
+We also strive to use this to highlight tools that share a common set of
+recommended characteristics:
+
+-  tools should include a description and an author
+-  tools should include a README.md or similar in their source repo
+   describing any other relevant information about the tool
+-  tools should include at least one test parameter file indicating how
+   to run the tool on some sample data
+-  the Dockerfile should be helpful in reconstructing how a tool was
+   built from source
+-  tools and/or their reference data should be publically available
+
+
+Why would I want to verify?
+-------------------------------------------
+There are several reasons why you would want a tool/workflow to be verified.
+If you're a platform owner, verifying would indicate to others that your platform is compatible with many tools/workflows on Dockstore so others will be more likely to use your platform.
+If you're tool/workflow developer, verifying would assure others that your tool/workflow is of high quality and is very likely to work for others.
+
+
+How do I verify?
+---------------------------------------
+If you are an admin/curator, follow the :ref:`Verification Process` section in this document.
+If you are not an admin/curator, please send us a heads-up via our GitHub issues or Gitter!
+
+
+How do I tell if a tool/workflow is verified?
+---------------------------------------------
+
+There are 3 new indicators on Dockstore.org that indicates whether or
+not the tool/workflow is verified.
+
+First go to the page of the tool/workflow such as
+https://dockstore.org/containers/quay.io/briandoconnor/dockstore-tool-md5sum:1.0.4?tab=info.
+Since this tool/workflow is verified, 3 indicators can be
+seen:
+
+.. figure:: /assets/images/docs/verification/tool.png
+   :alt: Tool Page
+
+   Tool Page
+
+1. At the top left, the checkmark indicates that at least one of the
+   tool/workflow’s version has been verified. As a whole, this
+   tool/workflow is considered verified.
+
+2. At the top right, the recent versions of the tool/workflow are
+   listed. There is a checkmark if a specific version of the
```

Comment:
I would say `... a checkmark for each specific version...`, but see how that sounds for you.